addons/website/static/lib/multirange/multirange_custom.js

```diff
@@ -295,11 +295,18 @@ class Multirange {
     }
 
     formatNumber(number) {
-        number = String(number).split('.');
-        if (number[1] && number[1].length === 1) {
-            number[1] += '0';
+        const locale = document.querySelector("html").getAttribute("lang");
+        let formatedNumber;
+        // We use a try catch in case the lang is not supported by the browser.
+        try {
+            formatedNumber = number.toLocaleString(locale, {
+                minimumFractionDigits: 2,
+                maximumFractionDigits: 2,
+            });
+        } catch (e) {
+            console.error(`The locale ${locale} is not supported by your browser.`);
+            formatedNumber = number;
```

Comment:
If the format fails, how about a fallback to the previous code?
```suggestion
            formatedNumber = number[0].replace(/(?=(?:\d{3})+$)(?!\b)/g, ',') + (number[1] ? '.' + number[1] : '.00');
```